### PR TITLE
Add MTKS-MATEKH743

### PR DESCRIPTION
**Pull request by SteveCEvans**
@MATEKSYS Please review and confirm that this correctly enables all FC features.

Depends on betaflight/betaflight#11305

### CODE REVIEW

File: configs/default/MTKS-MATEKH743.config [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+resource SERVO 2 E06
+resource PPM 1 C07
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 D05
+resource SERIAL_TX 3 D08
+resource SERIAL_TX 4 B09
+resource SERIAL_TX 7 E08
+resource SERIAL_TX 8 E01
+resource SERIAL_TX 11 C06
```
Comment (MATEKSYS) [inline configs/default/MTKS-MATEKH743.config:26]: remove "resource SERIAL_TX 11 C06", add "resource SERIAL_TX 6 C06"

File: configs/default/MTKS-MATEKH743.config [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature LED_STRIP
+feature OSD
+
+# serial
+serial 5 64 115200 57600 0 115200
+serial 7 1024 115200 57600 0 115200
```
Comment (MATEKSYS) [inline configs/default/MTKS-MATEKH743.config:167]: remove serial 7 definition, leave it as default

File: configs/default/MTKS-MATEKH743.config (hunk 1 above)
Comment (SteveCEvans) [reply configs/default/MTKS-MATEKH743.config:26]: Done

File: configs/default/MTKS-MATEKH743.config (hunk 2 above)
Comment (SteveCEvans) [reply configs/default/MTKS-MATEKH743.config:167]: This is the ESC_SENSOR input. See http://www.mateksys.com/wp-content/uploads/2020/10/H743-SLIM_Wiring.jpg

**Review by MATEKSYS — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**